src/Components/Activities/Form/index.jsx

```diff
@@ -0,0 +1,195 @@
+// import React, { useEffect, useState } from 'react';
+import { Link, useParams, useLocation, useHistory } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+
+import styles from './form.module.css';
+import Button from '../../Shared/Button';
+import { Input, Textarea } from '../../Shared/Inputs';
+import ConfirmModal from '../../Shared/ConfirmModal';
+import ResponseModal from '../../Shared/ResponseModal';
+
+const Form = () => {
+  const [activity, setActivity] = useState({ name: '', description: '', isActive: false });
+  const [confirm, setConfirmModal] = useState(false);
+  const [response, setResponseModal] = useState({ show: false, state: '', message: '' });
+  const { id } = useParams();
+  const location = useLocation();
+  const history = useHistory();
+
+  const redirectAfterSubmit = {
+    pathname: '/activities',
+    state: {
+      state: '',
+      message: ''
+    }
+  };
+
+  const handleConfirm = () => {
+    setConfirmModal(!confirm);
+  };
+
+  const handleResponse = (state, message) => {
+    setResponseModal({ ...response, show: !response.show, state, message });
+
+    setTimeout(() => {
+      setResponseModal({});
+    }, 2500);
+  };
+
+  useEffect(() => {
+    if (location.pathname.includes('edit')) {
+      const setupForm = async () => {
+        const { name, description, isActive } = await getActivity(id);
+        setActivity({ name, description, isActive });
+      };
+      setupForm();
+    }
+  }, []);
+
+  const getActivity = async (id) => {
+    try {
+      const res = await fetch(`${process.env.REACT_APP_API_URL}/api/activity/${id}`);
+      const { data } = await res.json();
+      return data;
+    } catch (error) {
+      console.log(error);
```

Comment:
Please delete the console.log. You can use also the shared modals to show the error (suggestion)